system_tests/test_happybase.py

```diff
@@ -877,3 +880,26 @@ def test_counter_dec(self):
         self.assertEqual(
             table.row(ROW_KEY1, columns=[COL1]), {COL1: _PACK_I64(-dec_value)}
         )
+
+
+class TestTable_region(BaseTableTest):
+    def test_region(self):
+        from google.cloud.bigtable.table import Table as _LowLevelTable
+
+        ALT_TABLE_NAME = "table_with_split_key"
```

Comment:
I think that it would be safer to add a unique id at the end of the table name `+ unique_resource_id("-")`.  That will prevent collisions in the future.